README.rst

```diff
@@ -0,0 +1,189 @@
+:warning:This PolicyStat branch readme. For official Readme please refer to `master branch <https://github.com/PolicyStat/ckeditor-dev/tree/master>`_:warning:
+
+##########################################
+CKEditor- The PolicyStat "Enhanced Editor"
+##########################################
+
+PolicyStat  uses `CKEditor <http://ckeditor.com/>`_
+as the successor/replacement to WYMeditor.
+
+********************************
+CKEditor Usage Within Policystat
+********************************
+
+.. _ckeditor-pages-using-ckeditor:
+
+Pages Using CKEditor
+====================
+
+Currently, CKEditor is enabled as the editor for the following pages:
+
+* Document clean
```

Comment:
This isn't the only page that CKeditor is enabled for. It should be active on the normal document edit and override pages too. Please verify.

Reply:
bump

Reply:
Yes. They are if CKEditor is enabled.

These templates contain CKEditor
- ./pstat/templates/document_control/document_admin_override_ckeditor.haml ✔️ 
- ./pstat/templates/document_control/document_edit_ckeditor.haml ✔️ 
- ./pstat/templates/document_import/clean_import_ckeditor.haml
- ./pstat/templates/editor_sandbox/editor_sandbox.haml 
- ./pstat/templates/import_translation/test_cleaning_upload.haml